inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/broadcast_power.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include <vector>
+#include <string>
+#include <ie_precision.hpp>
+#include <subgraph_tests/broadcast_power.hpp>
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+        InferenceEngine::Precision::FP32,
+        InferenceEngine::Precision::FP16,
+};
+
+const std::vector<std::map<std::string, std::string>> configs = {
+        {
+                {"GNA_DEVICE_MODE", "GNA_SW_EXACT"},
```

Comment:
If you don't define any scale factor, the default one will be 1.0f. Isn't better to define an input scale factor according to input data?